### ISSUE TITLE
Error opening with Eagle

### ISSUE THREAD

**Issue by FedorAdarichev** [vedderb/SparkSwitch#2]
Hello,

I am attempting to open SparkProtectionSwitch.sch with Eagle 8.4.1 and it is giving me the following message:

Error:

line 1, column 1: Start tag expected.

Any idea what I could be doing wrong?

Thanks!

**Comment by CrisAdam** [comment]
I don't think this was done in eagle, I think it was done in KiCad & thus probably won't open correctly with anything else
